samples/musicfestival-frontend-react/src/helpers/urlHelper.ts

```diff
@@ -14,10 +17,18 @@ const getImageUrl = (path = "") => {
     return path.startsWith("http") ? path : siteUrl + path
 }
 
-const extractParams = (urlPath: string) => {
+const extractParams = (token: string, urlPath: string) => {
+    var payload: Payload = {
+        c_id: undefined,
+        c_ver: undefined
+    };
```

Comment:
since c_id and c_ver are optional, seems we don't need to initialize them to undefined, they are undefined by default.

```suggestion
    var payload: Payload = {}
```